tools/publish-lib.mjs

```diff
@@ -158,22 +158,37 @@ try {
 execSync(`npm publish --access public --tag ${tag} --dry-run ${dry}`);
 
 function getDevVersion(potentialVersion) {
-  const result = JSON.parse(execSync(`npm view ${PREFIX}-${name} versions --json`).toString());
-  const lastVersionToIncrement = result
-    .filter(ver => ver.startsWith(mainPackageJson.version))
-    .map(ver => ver.match(/\d+$/)?.[0])
-    .filter(Boolean)
-    .map(ver => parseInt(ver, 10))
-    .sort((a, b) => a - b)
-    .reverse()[0];
-
-  if (typeof lastVersionToIncrement !== 'undefined') {
-    const incrementedNum = lastVersionToIncrement + 1;
-    potentialVersion = `${mainPackageJson.version}.${incrementedNum}`;
-  } else {
-    potentialVersion = `${mainPackageJson.version}.0`;
+  try{
```

Comment:
wrap only `const result = JSON.parse(execSync(`npm view ${PREFIX}-${name} versions --json`).toString());` in try catch and in case of error just set result as empty array

Reply:
Changed as suggested.